docs/getting-started/endgame-sensor-FDA.asciidoc

```diff
@@ -76,8 +83,6 @@ image::images/fda/select-fda.png[Select Full Disk Access]
 . Click the *+* button to view *Finder*. Find  `com.endgame.systemextension` and select it.
 . Continue below to enable Full Disk Access for the Endgame sensor.
 
-
-
 [discrete]
 [[endpoint-endgame-sensor]]
 == Enable Full Disk Access for the Endgame sensor
```

Comment:
Ricardo will be resisting step 5 in this section.

Reply:
[ecca7f8](https://github.com/elastic/security-docs/pull/3565/commits/ecca7f8b2a72922deb707a39c0baf091ca0eabd0) contains the input @ricardoungureanu provided via Slack.